### DISCUSSION TITLE
Issue upgrading Serverpod

### DISCUSSION THREAD

**Discussion by eamnicoletti** [serverpod/serverpod#860] [Q&A] [answered]
I've try to upgrade my project to Serverpod 1.1.0 and got this issue into generated/protocol.dart: 

> Missing concrete implementation of 'SerializationManagerServer.getTargetDatabaseDefinition'. Try implementing the missing method, or make the class abstract.

As a generated file, I cannot modify. I did `Flutter clean` and `Dart pub get` already. Does anyone have a solution?

**Comment by yahu1031** [comment]
Can you please provide a little more info? And Provide all the serverpods info.

**Comment by andriyndev** [comment, marked as answer]
Check if all the Serverpod libraries' versions are the same, specify the latest version explicitly in pubspec.yaml. I had another similar error because the old version of some Serverpod libraries were used because the new version of the libraries use image 4.\*.\* library while my project had used image 3.\*.\* library and thus the Serverpod libraries that used image 4.\*.\* in the latest version weren't upgraded.

**Comment by eamnicoletti** [comment]
I went through the following steps again:
1 - `dart pub global activate serverpod_cli`
2 - updated all dependencies in pubspec.yaml files
3 - `flutter clean`
4 - `dart pub get`

... and it works :)

Thank you for your time, fellows

**Reply by vlidholt** [reply]
Yay!
